cms/migrations/0058_add_external_marketing_url_product.py

```diff
@@ -75,5 +75,7 @@ class Migration(migrations.Migration):
                 null=True,
             ),
         ),
-        migrations.RunPython(migrate_external_marketing_url, migrations.RunPython.noop),
+        # Commenting this because we won't need to run data migration after the data has been migrated
+        # The data migration was done in https://github.com/mitodl/mitxpro/pull/2628/
+        # migrations.RunPython(migrate_external_marketing_url, migrations.RunPython.noop),
```

Comment:
Do we need to comment this out as it won't have any impact I think?

Reply:
It will be on the fresh instances at least. Because of that same reason that the field would have been gone from the file and the migration if tries to access it will break on a fresh instances or those older ones that have existing course or program pages.

Reply:
@asadali145 Should i merge this?